lib/_http_outgoing.js

```diff
@@ -203,11 +199,10 @@ ObjectDefineProperty(OutgoingMessage.prototype, '_headerNames', {
       const headers = this[kOutHeaders];
       if (!headers)
         return;
-      const keys = ObjectKeys(val);
-      for (var i = 0; i < keys.length; ++i) {
-        const header = headers[keys[i]];
+      for (const key of ObjectKeys(val)) {
```

Comment:
We should not make this change for performance reasons.

Reply:
resolved in https://github.com/nodejs/node/pull/30958/commits/17049d56621869d8776a922d5e56b6c5ec35be7b